architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -216,6 +217,60 @@ status:
 ----
 ====
 
+[[image-stream-image]]
+=== Image Stream Image
+
+Image Stream Image is a virtual resource that allows to retrieve an image from
+a particular _image stream_ where it is tagged. It's often abbreviated as
+_isimage_. It consists of two parts delimited by `@`: `<image stream
+name>@<image name>`. To refer to the image in the
+xref:image-stream-object-definition[example above], the _isimage_ would look
+like this:
+
+----
+origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
+----
+
+Users, without a permission to read or list images on the cluster lever, can still
+retrieve the images tagged in a project, they have access to, using this
+resource.
+
+[[image-stream-tag]]
+=== Image Stream Tag
+
+An _image stream tag_ is a named pointer to an image in an _image stream_. It
+is often abbreviated as _istag_. It can reference any local or externally
+managed image. It contains a history of images which is a stack of all images
```

Comment:
It contains a history of images represented as a stack of all images the tag ever pointed to.